mindone/models/modules/pos_embed.py

```diff
@@ -25,6 +25,14 @@ def get_2d_sincos_pos_embed(embed_dim: int, nh: int, nw: Optional[int] = None) -
     return pos_embed
 
 
+def get_1d_sincos_temp_embed(
```

Comment:
add docstring, and type hint of return type

Reply:
Fixed~